ansible/roles/session-valkey/templates/values.yml.j2

```
@@ -0,0 +1,54 @@
+global:
+  defaultStorageClass: '{{ SC_DEFAULT_STORAGE_CLASS_NAME }}'
+replica:
+  replicaCount: '{{ SESSION_VALKEY_REPLICAS }}'
```

remove the ' form the line and then the variable will be an int and no string
```suggestion
  replicaCount: {{ SESSION_VALKEY_REPLICAS }}
```